x-pack/plugins/security_solution/public/landing_pages/pages/translations.ts

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
Some considerations:
- Some translations are duplicated, they already exist at `x-pack/plugins/security_solution/public/app/translations.ts`, It would be better to use those ones instead.
- The `*_DESCRIPTION` translations will be defined in the new configs once we have them, so maybe it is better to just create the translation in the component itself by now, so we can copy/paste all the `const items = [..]` to the new configs directly.
- We can remove all `*_IMAGE_ALT` translation and have only one interpolated translation (as described in the previous comment) 
So all these translations could be reduced to only one entry.